src/apidocs/ApiDocsArea.tsx

```diff
@@ -224,4 +221,18 @@ const BaseClasses = ({ value }: { value: ApiDocsBaseClass[] }) => {
   );
 };
 
+const DocString = ({ value }: { value: string }) => {
+  const html = renderMarkdown(value);
+  console.log(value);
```

Comment:
Debug.

Something odd is going on. Most markdown is rendering OK, but the bold text is being returned with escaped asterisks. Perhaps this needs a change on the Pyright end. I can't figure out why this would happen though.

It's fine for the same documentation for signature help.

Reply:
Maybe I'm double-converting it to markdown on the sever side?

Reply:
Nope, this is just something that isn't supported in Pyright's doc string conversion.

![Screenshot 2021-09-30 at 17 31 43](https://user-images.githubusercontent.com/44397098/135495129-39a29bee-b3d9-4292-b937-eabb3212f728.png)

Reply:
I've hacked this for bold which we have a lot of, but it's clearly a hack as it has to work with escaped asterisks.

To fix this properly we'll need to spend some time with Pyright's docStringConversion code which will take a while to get into.

I've been giving some thought as to whether I should abandon the reSt -> markdown (in Pyright) -> HTML approach, but I think it's probably worth trying to fix these bugs as they'll lead to a better VS Code experience too. Even if we just end up tailoring the content somewhat.

I'll raise issues once this code ends up on main.